src/main/resources/application.yml

```diff
@@ -116,7 +116,7 @@ nomis:
     authentication-scheme: header
     client-authentication-scheme: header
   health.timeout: 1s
-  endpoint.timeout: 5s
+  endpoint.timeout: 3m
```

Comment:
this will increase the timeout on all requests to nomis user roles - probably best to have a separate webclient config for the download request?